src/Common/Commands.Common.Authentication.Test/Commands.Common.Authentication.Test.csproj

```diff
@@ -16,7 +17,8 @@
     <TargetFrameworkProfile />
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
     <RestorePackages>true</RestorePackages>
-    <NuGetPackageImportStamp>9fdcb6f4</NuGetPackageImportStamp>
```

Comment:
Is there a reason you are deleting this? If so, you likely should just remove the surrounding <NugetPackageImportStamp> tags.